tap_snowflake/sync_strategies/common.py

```diff
@@ -199,6 +204,10 @@ def row_to_singer_record2(catalog_entry, version, row, columns, time_extracted):
                 rec[column] = boolean_representation
             else:
                 rec[column] = elem.hex()
+        
+        # orjson is used when writing singer record and it does not recognize Decimal type
+        elif isinstance(elem, Decimal):
+            rec[column] = float(elem)
```

Comment:
Underlying issue is that `orjson` does not support Decimal, which is used by `pipelinewise-singer-python` package. 
We noticed that we are using double for representing number (sparkSchema), so converting to float made sense for us. @kchan-varicent